lib/game.rb

```diff
@@ -0,0 +1,72 @@
+class Game
+  attr_accessor :board, :player_1, :player_2
+  WIN_COMBINATIONS = [[0,1,2],[3,4,5],[6,7,8],[0,3,6],[1,4,7],[2,5,8],[0,4,8],[6,4,2]]
+
+  def initialize(player_1=Players::Human.new("X"), player_2=Players::Human.new("O"),board=Board.new)
+    self.board = board
+    self.player_1 = player_1
+    self.player_2 = player_2
+  end
+
+  def current_player
+    number_of_turns = self.board.cells.select {|turns| turns != " "}.count
+    if number_of_turns.even?
+      player_1
+    else
+      player_2
+    end
+  end
+
+  def won?
+    possible_winning_combination = WIN_COMBINATIONS.select do |combo|
+      turn_1, turn_2, turn_3 = board.cells[combo[0]], board.cells[combo[1]], board.cells[combo[2]]
+      turn_1 != " " && turn_2 == turn_1 && turn_3 == turn_2
+    end
+    if possible_winning_combination == []
+      false
+    elsif possible_winning_combination.count > 1
+      possible_winning_combination.select do |combo|
+        board.cells.min_by {|token| board.cells.count(token)} == board.cells[combo[0]]
+      end.flatten
+    else
+      possible_winning_combination.flatten
+    end
+  end
+
+  def winner
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.